### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by soulemanou-software**
###
- I styled the hold project back just to give it a professional look
- I added a personalised messaged box

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      rel="stylesheet"
+    />
+    <title>LeaderBoards</title>
+  </head>
+  <body class="flex">
+    <main class="container flex">
+      <h1>
+        <i class="bx bxs-star"></i><i class="bx bxs-star"></i
+        ><i class="bx bxs-star"></i> LeaderBoard
+      </h1>
```
Comment (codepantha) [inline src/index.html]: - [ ] In the wireframe layout, the "Leaderboard" heading is all the way to the left but yours is centered. Kindly adjus it to match the wireframe layout
### Your Layout
image
### Wirefame layout
image

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          <div class="score-display">
+            <h2 class="notif">Refresh to load data</h2>
+          </div>
+        </div>
+
+        <form>
+          <h2 class="score-header border-radius">Add Your Score</h2>
+          <input type="text" placeholder="Enter Your Name" id="name" />
+          <input type="number" placeholder="Enter Your Score" id="score" />
+          <button type="submit" id="submit">Submit</button>
```
Comment (codepantha) [inline src/index.html]: - [ ] Please align your "submit" button to the right so that your layout matches the wireframe 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @soulemanou-software 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Beautiful and responsive UI
✔️ Descriptive PR
✔️ NO linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Pull Request :red_circle: :traffic_light: :rotating_light:

## Hey @soulemanou-software :wave: 

**You have made a great effort on your project :blush: and I can see that but unfortunately I can not proceed with a code review of your pull request due to the following reasons :receipt:**

## 1. 
- [ ] Please fix your linter erros because linters prevent you from making those errors which can  break your application :fearful:
### Once you implement all the changes you should be ready to submit your project for a review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @soulemanou-software , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great UI design :heavy_check_mark: 
- Good project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**